classes/metric/builtin.php

```diff
@@ -47,7 +47,7 @@ public function get_label(): string {
      * @return int
      */
     public function get_frequency(): int {
-        return (int) get_config('tool_cloudmetrics', 'newusers_frequency');
+        return (int) get_config('tool_cloudmetrics', $this->get_name() . '_frequency');
```

Comment:
This feels like it should be in the base class, so you dont need to override it again in non-builtins

Reply:
I don't want to make any assumptions about how a plugin may handle the internals of it's metrics. Nor do I want to enforce anything onto them other than the interface itself.

Reply:
This can just be overridden though. Anyone who wants a different frequency definition can, but it saves copy pasting when 99% of cases will remain the same

Reply:
Done